### DISCUSSION TITLE
Load model, without having "mu" parameter

### DISCUSSION THREAD

**Discussion by ssaha1234** [scikit-hep/pyhf#1752] [Q&A] [answered]
Dear pyhf community, 
    I am performing a Control Region only fit for my analysis. I have the likelihood with all the regions, but I do not have `mu` parameter defined under `channels`, as I am assuming Signal = 0 in the CRs. I have `mu` defined under `measurements` as the `poi`. 

I am processing the following lines for the mle fit -
 
`wspace = pyhf.Workspace(json.load(open(sys.argv[1])))`
`model = wspace.model()`
`data = wspace.data(model)`
`init_pars = model.config.suggested_init()`
`fixed_params = model.config.suggested_fixed()`

`result, result_obj = pyhf.infer.mle.fit(
    data,
    model,
    maxiter=1000000,
    init_pars=init_pars,
    fixed_params=fixed_params,
    return_uncertainties=True,
    return_result_obj=True
)
`

At the moment, the code crashes when I try to load the model in this line - `model = wspace.model()` as `mu` is not defined. I am thinking if I can forcefully set `mu` to 0, to do the fit, without the crash ? The fit works fine when include all the regions (SR+CR). 

Thanks ! 
Best, 
Shreya

**Comment by alexander-held** [comment, marked as answer]
Hi @ssaha1234, setting `mu` to a constant value of 0 will not be enough to avoid that crash I think. You would additionally need to keep some sample in the model on which `mu` acts. Since `mu` will be constant 0, that sample effectively does nothing and this should result in the model you are looking for I believe.

A more convenient method has been implemented after `pyhf` version `0.6.3` was released, see #1602 / #1636 / #1638. It allows you to set `"poi": ""` in the config, or use `wspace.model(poi_name=None)` for fits without POI. You can already use `poi_name=None` now (thanks to #950), however you will not get through model validation without the workaround shown at the bottom of https://github.com/scikit-hep/pyhf/discussions/1602#discussion-3583594. That workaround may be the most convenient thing to use with `pyhf` version `0.6.3`, and `0.7.0` will bring the new options.

**Reply by ssaha1234** [reply]
Hi @alexander-held, 
   Thank you very much for the detailed workarounds, I am looking through the tickets and comments, and I will try it out after ! 
Best, 
Shreya
